content/en/docs/languages/java/automatic/configuration.md

```diff
@@ -228,6 +228,59 @@ span link connecting it to the producer trace.
 > **Note**: The property/environment variable names listed in the table are
 > still experimental, and thus are subject to change.
 
+### Capturing Enduser Attributes
+
+You can configure the agent to capture
+[general identity attributes](/docs/specs/semconv/general/attributes/#general-identity-attributes)
+(`enduser.id`, `enduser.role`, `enduser.scope`) from instrumentation libraries
+like
+[JavaEE/JakartaEE Servlet](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/servlet)
+and
+[Spring Security](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring/spring-security-config-6.0).
+
+This functionality is disabled by default, with the ability to enable for
+individual attributes.
```

Comment:
please call out that they are disabled by default because of privacy concerns, and that people should be careful and verify that they are even allowed to turn this on